modules/EnsEMBL/Web/Apache/Handlers.pm

```diff
@@ -109,9 +109,9 @@ sub get_redirect_uri {
     return $uri =~ s/\/Multi\/psychic/\/Multi\/Psychic/r;
   }
   
-  ## Redirect incoming search links from ensemblgenomes.org 
-  if ($uri =~ /\/common\/psychic/) {
-    return $uri =~ s/\/common\/psychic/\/Multi\/Psychic/r;
+  ## Handle id lookups from ensemblgenomes.org to the corresponding page
```

Comment:
Are you 100% sure we can remove these lines ?

Reply:
Yes, Steve. I added this line in my previous PR here: 
https://github.com/Ensembl/ensembl-webcode/pull/750/files#diff-dc4adb54dad99daa9c4d1fb60ed7dc74R114 

Previous PR fixed the not found error and redirected the ID lookups to search page. This PR will directly redirect the lookups to the actual summary page itself.